ansible/roles/kong-consumer/defaults/main.yml

```diff
@@ -43,6 +43,7 @@ kong_all_consumer_groups:
   - contentAdmin
   - contentCreate
   - contentUpdate
+  - contentSuperAdmin
```

Comment:
we should never give access to private apis to the appliaction, it is against the standards